### DISCUSSION TITLE
[Bug Report] vant4开发uniapp app-plus端 报getComputedStyle错误

### DISCUSSION THREAD

**Discussion by yqzyou** [youzan/vant#12231] [Q&A]
### 重现链接

https://vant-contrib.gitee.io/vant/#/zh-CN/list

### Vant 版本

4.1.2

### 描述一下你遇到的问题。

uniapp app-plus 端使用van-list、van-action、showImagePreview这些组件，报错误信息如下：

`Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'getComputedStyle')
    at getScrollParent$1 ()
    at Object.<anonymous> ()
`
### 重现步骤

1. 页面直接引用了van-list等组件
2. 页面初始化加载，就报错了

### 设备/浏览器

ios、android

**Comment by opendidi** [comment]
请问一下解决了嘛？

**Reply by wowoName** [reply]
拉  解决了吗

**Comment by kgdusouihcy** [comment]
还没解决，这个问题是vant库使用了window对象，而uni-plus就没有window对象
